frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNoteEditor.tsx

```diff
@@ -0,0 +1,288 @@
+          paddingVertical={'zero'}
+        >
+          <BackButton
+            onClick={() => history.goBack()}
+            icon={faArrowLeft}
+            text={
+              child ? `${child.firstName} ${child.lastName}` : i18n.common.back
+            }
+          />
+          <ContentAreaWithShadow
+            opaque={true}
+            paddingHorizontal={'s'}
+            paddingVertical={'m'}
+          >
+            <Title>{i18n.attendances.notes.dailyNotes}</Title>
+            <FixedSpaceColumn spacing={'m'}>
+              <FixedSpaceColumn spacing={'xxs'}>
+                <Label>{i18n.attendances.notes.labels.note}</Label>
+                <InputField
+                  value={dailyNote.note}
+                  onChange={(value) =>
+                    setDailyNote({ ...dailyNote, note: value })
+                  }
+                  placeholder={i18n.attendances.notes.placeholders.note}
+                />
+              </FixedSpaceColumn>
+
+              <FixedSpaceColumn spacing={'s'}>
+                <Label>{i18n.attendances.notes.labels.feedingNote}</Label>
+                <FixedSpaceColumn spacing={'xs'}>
+                  {levelInfoValues.map((levelInfo) => (
+                    <Radio
+                      key={levelInfo}
+                      label={i18n.attendances.notes.values[levelInfo]}
+                      onChange={() =>
+                        setDailyNote({ ...dailyNote, feedingNote: levelInfo })
+                      }
+                      checked={dailyNote.feedingNote === levelInfo}
+                      data-qa={`feeding-note-${levelInfo}`}
+                    />
+                  ))}
+                </FixedSpaceColumn>
+              </FixedSpaceColumn>
+
+              <FixedSpaceColumn spacing={'s'}>
+                <Label>{i18n.attendances.notes.labels.sleepingNote}</Label>
+                <FixedSpaceColumn spacing={'xs'}>
+                  {levelInfoValues.map((levelInfo) => (
+                    <Radio
+                      key={levelInfo}
+                      label={i18n.attendances.notes.values[levelInfo]}
+                      onChange={() =>
+                        setDailyNote({ ...dailyNote, sleepingNote: levelInfo })
+                      }
+                      checked={dailyNote.sleepingNote === levelInfo}
+                      data-qa={`sleeping-note-${levelInfo}`}
+                    />
+                  ))}
+                </FixedSpaceColumn>
+              </FixedSpaceColumn>
+
+              <FixedSpaceColumn spacing={'s'}>
+                <Label>{i18n.attendances.notes.labels.reminderNote}</Label>
+                <FixedSpaceColumn spacing={'xs'}>
+                  {reminderValues.map((reminder) => (
+                    <Checkbox
+                      key={reminder}
+                      label={i18n.attendances.notes.reminders[reminder]}
+                      onChange={() => {
+                        dailyNote.reminders.includes(reminder)
```

Comment:
Can't you just use the `checked` boolean given to `onChange` as an argument? Or am I not seeing something?

Reply:
This was just some copypaste code. Tried with the `checked` and it was neater (and more performant!)!